app/Models/User.php

```diff
@@ -30,6 +30,7 @@
  * @property string|null                                           $email
  * @property bool                                                  $may_upload
  * @property bool                                                  $is_locked
+ * @property string                                                $token
```

Comment:
Even though the token is public information (is it?) I wonder, if we should it to the `hidden` array such that it is not sent to the client when the user object is converted into JSON.